app/scripts/controllers/reviewRegistration.js

```diff
@@ -249,8 +249,13 @@ angular.module('confRegistrationWebApp')
     $scope.mergeAndConfirmRegistration = function () {
       // Pay for the spouse's registration before merging it with the spouse
       $scope.submittingRegistration = true;
-      registration.submitPayment($scope.currentPayment, $scope.spouseRegistration, $scope.acceptedPaymentMethods()).then(function () {
-        return registration.mergeWithSpouse(currentRegistration, $scope.spouseRegistration);
+
```

Comment:
@hlbraddock Ok. Just a thought I had.  I do see there is a validate method in the Payment API.  Below is from createPayment.  Wondering what could be causing the spouse payment to invalidate.
```
155 		payment.validate(); 
156 
 
157 		// check and make sure the payment amount is valid, but allow administrators to do whatever amounts they want 
158 		if(!isPaymentAmountValid(payment.getAmount(), loggedInUser, registrationEntity)) 
159 		{ 
```
https://github.com/CruGlobal/ert-api/blob/master/src/main/java/org/cru/crs/api/PaymentResource.java